x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/user_agent.tsx

```diff
@@ -78,7 +95,39 @@ export const UserAgent: FunctionComponent = () => {
           'xpack.ingestPipelines.pipelineEditor.userAgentForm.propertiesFieldHelpText',
           { defaultMessage: 'Properties added to the target field.' }
         )}
-        propertyOptions={propertyOptions}
+        euiFieldProps={{
+          options: propertyOptions as [],
+          noSuggestions: !propertyOptions,
```

Comment:
Can this be set to `false`? Won't `propertyOptions` always be defined?

Reply:
yeah you are right, I was just refactoring these bits so didnt look at it too much. Will just set it to false!